app/controllers/departments_controller.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+class DepartmentsController < ApplicationController
```

Comment:
Move this into the `app/controllers/org_admin` dir and the `OrgAdmin` namespace. Then remove the `admin_` prefix from the action names so that we just have: `new`, `update`, etc. 

The other `admin_` prefixed controller actions out there are legacy code we never finished cleaning up.

Reply:
it would be a good bit of cleanup to go through and re-factor those to the org_admin namespace next sprint.